views/index-getInvolved.html

```diff
@@ -1,5 +1,6 @@
-<div class="getInvolvedSection container">
+<div class="getInvolvedSection">
   <h2>Get Involved</h2>
+  <div class="container">
 
   <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
```

Comment:
If you are going to add a new `div`, you ought to "tab" over the inner content for legibility.  Right now you have...
```
<div class="container">
<div></div>
<div></div>
<div></div>
</div>
```
But you ought to tab it over by two spaces
```
<div class="container">
  <div></div>
  <div></div>
  <div></div>
</div>
```